hamlet-cli/README.md

```diff
@@ -1,5 +1,84 @@
-# Hamlet Deploy - Cli
+# Hamlet Deploy - CLI
 
-This is a python-based CLI for the Hamlet Deploy application. It acts as the primary interface to Hamlet Deploy, its Plugins and Extensions.
+Hamlet deploy is a tool to manage infrastructure throughout the life of your application. With hamlet you define the functional components of your application along with the context they should be run in. The context includes things like environments, tenants, and policies that can be applied across all of your different applications.
+From this information hamlet then creates the infrastructure that will perform the function you have asked for and manages it over the life of your application.
 
-See https://docs.hamlet.io for more info on Hamlet Deploy
+## Support
+
+Hamlet deploy is built on a plugin based approach so you can create infrastructure for any provider you want, we provide an API of common processes and tasks for provisioning and describing infrastructure which allows you to build a common language across different infrastructure providers
+
+We also provide official plugins for:
+
+- Amazon Web Services (AWS) based on CloudFormation
+- Microsoft Azure based on Azure Resource Manager
+- mingrammer Diagrams for generating diagrams of your deployed infrastructure
+
+## Docs
+
+To read more about hamlet and what it can do head to our docs site https://docs.hamlet.io/
+
+## Installation
+
+Hamlet Deploy is made up of a couple of parts:
+
+- engine - The engine is the core of hamlet and defines how infrastructure is deployed along with understanding the context of your application defined in a CMDB. The engine creates contracts and supporting documents which describes a task to complete. The engine is a java based app using the freemarker template engine.
+- executor - The executor providers the user interface to hamlet along with the executing what the contracts provided by the engine. This is a mix of bash scripts and python code ( this cli )
+
+### Docker Image
+
+This guide takes you through installing the cli on your own machine and includes the OS packages that are required. We also offer a docker image which includes the hamelt cli along with a suite of tools to use the image as a general purpose CI/CD environment
+
+The image is available on Dockerhub - https://hub.docker.com/r/hamletio/hamlet
+
+### Prerequisites
+
+Before using the cli a few OS level requirements need to be installed. This process differs from OS to OS so we are just including the packages and the websites if they are available. The packages need to be available on the PATH of your cli
+
+#### Minimum
+
+These packages will give you base access to start using hamlet
+
+| Name   | Install Link                                 | Version           |
+|--------|----------------------------------------------|-------------------|
+| Java   | https://openjdk.java.net/install/            | 8/1.8 Required    |
```

Comment:
I think this now needs to be 11? Need to confirm with Kseniya if new wrapper still runs on 8

Reply:
Not just yet. The PR hasn't been tested and @kshychko has put it back to a draft this morning.